pg_diffix--0.0.1.sql

```diff
@@ -219,6 +219,12 @@ CREATE AGGREGATE is_suppress_bin(*) (
   initcond = false
 );
 
+CREATE FUNCTION internal_qual_wrapper(boolean)
+RETURNS boolean
+AS 'MODULE_PATHNAME'
+LANGUAGE C VOLATILE
```

Comment:
Maybe it could, but I was afraid of some hypothetical inlining or other static analysis. We really want to scare Postgres into thinking something volatile happens in this function...